src/helpers/removePages.js

```diff
@@ -0,0 +1,27 @@
+import core from 'core';
+import actions from 'actions';
+import i18next from 'i18next';
+
+export default pages => dispatch => {
```

Comment:
I think we can just move the logic into the thumbnail control component. I don't think we are reusing this function right?

Reply:
I removed the helper and moved all the logic to the thumbnail panel